src/eeprom.js

```diff
@@ -338,4 +338,8 @@ var defaultEepromFiles = {
     'test-cape.eeprom': {
         type: 'cape'
     },
-};
\ No newline at end of file
+};
+module.exports = {
+    readEeproms: readEeproms,
+    fillCapeEepromData: fillCapeEepromData
```

Comment:
FYI, this function needs some love and testing. See https://github.com/jadonk/beagle-tester for some more recent EEPROM programming code.

Reply:
I will definitely look more in to this part of the library.